package.json

```diff
@@ -27,6 +27,7 @@
     "@types/chai": "^4.1.2",
     "@types/mocha": "^2.2.48",
     "chai": "3.5.0",
+    "codecov": "^3.8.3",
```

Comment:
Hey it looks like this package has been deprecated 

https://www.npmjs.com/package/codecov

Reply:
Thanks for the heads up, I completely missed that.